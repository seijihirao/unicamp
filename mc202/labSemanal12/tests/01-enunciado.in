9
4 4 9
4 5 8
2 3 6
2 4 5
0 4 3
0 5 7
1 0 1
2 2 2
4 1 4
3 3
1 1
5 5
6 6
2 2
4 4
7 7
-1 -1
