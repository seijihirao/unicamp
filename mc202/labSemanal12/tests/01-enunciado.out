(3,3) = 0
(1,1) = 0
(5,5) = 0
(6,6) = 0
(2,2) = 2
(4,4) = 9
(7,7) = 0
